avoinspectorexample/src/main/java/app/avo/avoinspectorexample/KotlinActivity.kt

```diff
@@ -34,11 +35,11 @@ class KotlinActivity : Activity() {
                 "Float Name" to 1.0,
                 "Bool Name" to true))
 
-        avoInspector.showVisualInspector(this, DebuggerMode.bubble)
+        avoInspector.showVisualInspector(this, VisualInspectorMode.bubble)
 
         avoInspector.hideVisualInspector(this)
 
-        val visualInspector = avoInspector.visualInspector
+        val visualInspector = avoInspector.visualInspector as DebuggerManager
```

Comment:
Is this class a WIP (work in progress) or I'm missing something here? 
Currently the code in this class doesn't show me the proper way to use Avo nor it has some useful information / comments.

Reply:
Hm, right, would improve the examples in a separate PR